### PR TITLE
[tests-only]Add bug demonstration test for sending PUT request to other user's webdav endpoint

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR adds bug demonstration tests for sending PUT request to other user's webdav endpoint

## Related Issue

- Bug demo for : https://github.com/owncloud/core/issues/39597

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiAuthWebDav/webDavPUTAuth.feature [hunk 1]
```diff
@@ -36,8 +36,9 @@ Feature: get file info using PUT
       | /remote.php/dav/files/%username%/PARENT/parent.txt |
     Then the HTTP status code of responses on all endpoints should be "401"
 
-  @issue-ocis-reva-9 @issue-ocis-reva-197 @skipOnOcV10.3 @skipOnOcV10.4 @skipOnOcV10.5
-  Scenario: send PUT requests to another user's webDav endpoints as normal user
+  @issue-39597 @skipOnOcV10.3 @skipOnOcV10.4 @skipOnOcV10.5 @skipOnOcis
+    #After the issue-39597 is fixed use the scenario below
+  Scenario: send PUT requests to another user's webDav endpoints as normal user (bug demostration)
```
Comment (phil-davis) [inline tests/acceptance/features/apiAuthWebDav/webDavPUTAuth.feature:41]: Nowadays we move the oC10 bug demonstration scenarios into a separate feature file. The advantage is that when the bug is fixed, the developer can delete the separate feature file, and just remove the `skipOnOcV10` tag from the good scenario in the "real" feature file. No "real" line numbers change in the "real" feature file.

Have a look at `apiProvisioningGroups-v1/addGroupOc10Issue31015.feature` for an example.

**Review by phil-davis — APPROVED** [review]
LGTM. The "good" test is the way that all implementations should behave (403 status)

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIMobileSize-3-1-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34036/167/1

**Comment by phil-davis** [comment]
I have cherry-picked this to #39514 where we are putting test changes while release-10.9.0 is happening.

**Comment by SwikritiT** [comment]
> I have cherry-picked this to #39514 where we are putting test changes while release-10.9.0 is happening.

I'll bump the commit id's now.

**Comment by phil-davis** [comment]
CI passed. The code is in #39514 - closing here.

**Pull request closed without merging**